magento/magento2
# Feature bulgarian country regions

### Description (*)
Adding Bulgarian regions to **directory_country_region** table


### Related Pull Requests


### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Bulgarian states.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#28215: Feature bulgarian country regions
